src/common-principles.md

```diff
@@ -536,12 +550,39 @@ like in the example below.
             "F": {
                 "Description": "Female",
                 "TermURL": "https://www.ncbi.nlm.nih.gov/mesh/68005260"
-            },
+            }
         }
     }
 }
 ```
 
+### Compressed tabular files
+
+Large tabular information such as physiological recordings MAY be stored with
+[compressed tab-delineated (TSVGZ) files](glossary.md#tsvgz-extensions).
```

Comment:
IMHO ambiguous whether this allows TSVGZ for ALL TSV files (xref: https://github.com/bids-standard/bids-specification/pull/1749#issuecomment-2021000568)

Reply:
```suggestion
Large tabular information, such as physiological recordings, SHOULD be stored with
[compressed tab-delineated (TSVGZ) files](glossary.md#tsvgz-extensions) when
so established by the specifications.
```